src/components/Reports/GoalCalculator/SharedComponents/GoalCalculatorGrid/GoalCalculatorGrid.tsx

```diff
@@ -196,7 +196,7 @@ const GoalCalculatorGridForm: React.FC<GoalCalculatorGridFormProps> = ({
   const columns: GridColDef[] = [
     {
       field: 'name',
-      headerName: t(headerName),
+      headerName: t(categoryName),
```

Comment:
`t` needs to be passed a static string, not a variable so that `yarn:extract` can search the codebase for all of our labels. We should make sure that `categoryName` is localized where it is passed in and then remove this `t()` call.